src/phoenix/trace/dsl/helpers.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Optional, Protocol, Union, cast
+
+import pandas as pd
+
+from phoenix.trace.dsl import SpanQuery
+from phoenix.trace.semantic_conventions import (
+    DOCUMENT_CONTENT,
+    DOCUMENT_SCORE,
+    INPUT_VALUE,
+    OUTPUT_VALUE,
+    RETRIEVAL_DOCUMENTS,
+)
+
+INPUT = {"input": INPUT_VALUE}
+OUTPUT = {"output": OUTPUT_VALUE}
+IO = {**INPUT, **OUTPUT}
+
+IS_ROOT = "parent_id is None"
+IS_LLM = "span_kind == 'LLM'"
+IS_RETRIEVER = "span_kind == 'RETRIEVER'"
+
+
+class Session(Protocol):
+    def query_spans(self, *query: SpanQuery) -> Optional[Union[pd.DataFrame, List[pd.DataFrame]]]:
+        ...
+
+
+def get_retrieved_documents(session: Session) -> pd.DataFrame:
```

Comment:
need to be exported at a lower level in the module tree `trace.dsl.helpers` is too deep. Maybe session evals like you have

Reply:
ok, i'll just re-export it over there for now before we find a final home